formidling/src/main/java/no/nav/ung/sak/formidling/VedtaksbrevRegler.java

```diff
@@ -122,6 +151,14 @@ private VedtaksbrevRegelResulat lagResultatMedBygger(Behandling behandling, Loca
         return VedtaksbrevRegelResulat.ingenBrev(detaljertResultat, forklaring);
     }
 
+    private boolean erFørsteOpphør(Behandling behandling) {
+        var forrigeGrunnlag = ungdomsprogramPeriodeRepository.hentGrunnlag(behandling.getOriginalBehandlingId().orElseThrow(
+            () -> new IllegalStateException("Må ha original behandling ved opphør")
+        )).orElseThrow(() -> new IllegalStateException("Mangler grunnlag for forrige behandling"));
+        return forrigeGrunnlag.getUngdomsprogramPerioder().getPerioder().stream()
```

Comment:
@espenjv slik sjekker jeg om sluttdato er satt for første gang. Er det noe bedre måte å gjøre det på tror du?